sbin/pfdetect

```diff
@@ -156,10 +156,11 @@ sub _run_detector {
         my $line = $_;
         $logger->info("alert received: '$line'");
         my $data = $parser->parse($line);
-        unless($data) {
+        unless($data || $data eq 0) {
```

Comment:
Totally agree, but he was first in line at the coffee machine and I was eager to have my coffee so I accepted to make the modification in exchange of the first place.